### ISSUE TITLE
Doesn't work with Rails 5 and Spree 3.2.0

### ISSUE THREAD

**Issue by priom** [spree-contrib/spree_drop_ship#80]

**Comment by mayordwells** [comment]
I am currently using spree to create a marketplace app and It will be really nice if this could be upgraded to work with Rails 5 and Spree 3.2.0. Is there a possibility for that in coming weeks or months?

**Comment by bradpurchase** [comment]
+1

**Comment by haxplorer** [comment]
+1
Rails - 5.1
Spree - 3.3 and above

**Comment by MateoLa** [comment]
Hi. I've made some modifications in the gem to work with rails 5.1 and spree 3.4.4. You could find them in https://github.com/MateoLa/spree_drop_ship/releases v1-0-0.
Let me know if it is useful for someone.

**Comment by dbbooke** [comment]
@MateoLa How do I import the upgraded to gem into my application?

**Comment by bradpurchase** [comment]
@dbbooke We did it by pointing to @MateoLa's fork in our `Gemfile`, like this:

```
gem 'spree_drop_ship', git: 'https://github.com/MateoLa/spree_drop_ship'
```

**Comment by dbbooke** [comment]
I don't think you guys know how much the old gem provided by Spree-contrib has been bugging me because it didn't work lol. Appreciate it guys

**Comment by bradpurchase** [comment]
@dbbooke yeah, our Rails 5 upgrade has been delayed for months mostly because we didn't have the time to do this update ourself, so really appreciate it @MateoLa!

**Comment by dbbooke** [comment]
One more question, I am getting this error after migrating @MateoLa 
`WARN  DurableDecorator: Spree::StockLocation#propagate_variant decoration uses an invalid SHA. The original method definition could have been tampered with!
WARN  DurableDecorator: Expected SHA was 68fd322da48facbdbf567a2391c2495d04c3c8bc but the provided SHA is f35b0d8a811311d4886d53024a9aa34e3aa5f8cb`

**Comment by dbbooke** [comment]
Also, the Doc_ship gem is required for spree_marketplace gem. Is anyone else having a hard time with that gem with Rails 5?

**Comment by MateoLa** [comment]
Hi Darren and Brand.

I don't know if I'm going to need to work with spree-marketplace in the
near future. I let you know if I do it.
Regarding the SHA key error, replece the key in the correct place and the
error will go away.

I also have litle free time because I trying to adapt my site to go online.

I really appreciate your feedback.

Thanks, Mateo.



2018-04-03 17:38 GMT-03:00 Darren Booker <notifications@github.com>:

**Comment by mices** [comment]
Why is contrib's latest version only for spree 2.2 When are they gonna get caught up?
